### DISCUSSION TITLE
Add additional custom fields on post meta data

### DISCUSSION THREAD

**Discussion by harshvardhanbarhan** [austintoddj/canvas#1313] [General]
Hi Everyone
I want to add some more option to post meta data or post general settings
Is there any simple way to extend the meta or general setting modal

![image](https://github.com/austintoddj/canvas/assets/43598666/8175b200-144e-47c4-8119-c0ea5056a50b)

**Comment by austintoddj** [comment]
@harshvardhanbarhan What other fields would you like to see added?
